packages/lumx-react/src/components/text-field/TextField.tsx

```diff
@@ -71,13 +71,13 @@ interface TextFieldProps extends GenericProps {
     inputRef?: RefObject<HTMLInputElement> | RefObject<HTMLTextAreaElement>;
 
     /** Text field value. */
-    value: string;
+    value: string | number;
```

Comment:
I think we need this to be optional since for `number` value you may need to initialize the input as empty and the empty string is not very compatible with `number`
```suggestion
    value?: string | number;
```